circle.yml

```diff
@@ -0,0 +1,55 @@
+machine:
+  environment:
+    ARCH: "i686"
+    BUILDOPTS: "-j3 VERBOSE=1 FORCE_ASSERTIONS=1 LLVM_ASSERTIONS=1"
+    TESTSTORUN: "all"
+    JULIA_CPU_CORES: 2
```

Comment:
They have 2 vCPUs. But you can run up to 4 images in parallel in a single build and can manually split your tests across them. Not sure how helpful that will be for us given the current structure of the tests.